draft-ietf-core-dynlink.md

```diff
@@ -106,6 +105,7 @@ Conditional Control Attributes are defined below:
 | Attribute         | Parameter | Value            |
 | Minimum Period (s)| pmin      | xs:decimal (>0) |
 | Maximum Period (s)| pmax      | xs:decimal (>0) |
+| Change Step       | st        | xs:decimal (>0) |
 | Minimum Evaluation Period (s)| epmin      | xs:decimal (>0) |
 | Maximum Evaluation Period (s)| epmax      | xs:decimal (>0) |
 {: #weblinkattributes title="Conditional Control Attributes"}
```

Comment:
The Travis CI build is failing on this document owing to "weblinkattributes" being a duplicate label from the previous table. I think it's better to rename "weblinkattributes" on this line to, for example, "controlattributes".

Reply:
Should we change #weblinkattributes to #notificationattributes in the Notification Attributes as well?

Reply:
I can't find any documentation on this title format and it only displays as a table entry for me. I am using the browser for github access.